core/layout_api.php

```diff
@@ -613,15 +609,11 @@ function layout_navbar_projects_list( $p_project_id = null, $p_include_all_proje
 			$t_can_report = access_has_project_level( $t_report_bug_threshold, $t_id, $t_user_id );
 		}
 
-		echo '<li><a href="' . helper_mantis_url( 'set_project.php' ) . '?project_id=' . $t_id . '"';
+		echo 0 == strcmp( $t_id, $p_project_id ) ? '<li class="active">' : '<li>';
```

Comment:
@vboctor $t_id is int while $p_project_id is a string. The == check will succeed in pass for the top parent project (int comparison - skips text ;xx) and the selected subproject (string comparison). The correct behavior is single selection thus we need to enforce string comparison.